test_mem/test_mem_genesis2.py

```diff
@@ -99,10 +99,6 @@ def write(self, addr, data):
     def expect_read(self, addr, data):
         self.poke(self.circuit.clk_in, 0)
         self.eval()
-        # These values might change early, but we don't expect anything until
-        # after the 1 cycle read delay
-        self.expect(self.circuit.data_out, None)
-        self.expect(self.circuit.chain_out, None)
```

Comment:
Note I decided to remove these expects because they now default to None/X so we don't need to specify them explicitly.